va_explorer/va_analytics/dash_apps/assets/local.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function(){
```

Comment:
I don't see this file included anywhere. Does it get automatically loaded somehow or is it not used?